src/CMakeLists.txt

```diff
@@ -614,7 +614,9 @@ add_executable(ceph-objectstore-tool
   $<TARGET_OBJECTS:common_util_obj>)
 target_link_libraries(ceph-objectstore-tool osd os global ${Boost_PROGRAM_OPTIONS_LIBRARY} ${CMAKE_DL_LIBS})
 install(TARGETS ceph-objectstore-tool DESTINATION bin)
-
+if(WITH_FUSE)
+target_link_libraries(ceph-objectstore-tool fuse)
```

Comment:
indent this line if you please. and squash this commit into the previous one. then this PR is good to merge.